editor.planx.uk/src/@planx/components/Checklist/Public/Public.tsx

```diff
@@ -1,28 +1,156 @@
-import { useStore } from "pages/FlowEditor/lib/store";
-import React from "react";
+import Grid from "@mui/material/Grid";
+import { visuallyHidden } from "@mui/utils";
+import { checklistValidationSchema } from "@planx/components/Checklist/model";
+import Card from "@planx/components/shared/Preview/Card";
+import { CardHeader } from "@planx/components/shared/Preview/CardHeader/CardHeader";
+import { getIn, useFormik } from "formik";
+import { partition } from "lodash";
+import React, { useEffect } from "react";
+import FullWidthWrapper from "ui/public/FullWidthWrapper";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
+import { object } from "yup";
 
+import { Option } from "../../shared";
 import { Props } from "../types";
-import { AutoAnsweredChecklist } from "./components/AutoAnsweredChecklist";
-import { VisibleChecklist } from "./components/VisibleChecklist";
+import { ChecklistItems } from "./components/ChecklistItems";
+import { ExclusiveChecklistItem } from "./components/ExclusiveChecklistItem";
+import { GroupedChecklistOptions } from "./components/GroupedChecklistOptions";
+import { toggleNonExclusiveCheckbox } from "./helpers";
+import { useExclusiveOption } from "./hooks/useExclusiveOption";
+import { useSortedOptions } from "./hooks/useSortedOptions";
+
+export enum ChecklistLayout {
+  Basic,
+  Grouped,
+  Images,
+}
```

Comment:
I think this should be in model.ts or types.ts

Reply:
Yep good call - updated !